distribution/engine/THIRD-PARTY/org.graalvm.truffle.truffle-nfi-23.1.0/NOTICES

```diff
@@ -0,0 +1,7 @@
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
```

Comment:
These should not appear in the notices - this sentence is taken out of context and does not make any sense. Please ensure it is either put with whole context or removed, depending if it is needed.

Reply:
When I worked with the legal review tool, I clicked on this entry in the table, and it expanded many lines that did make sense - there were few copyright headers merged. So I figured that these copyright headers we want to keep in our notices. How do I put the whole context? I simply clicked on "Keep" button in the legal review tool.

Reply:
Fixed in 1ed637aeb465ddaf3ca5336327997703e3627a00